web_external/templates/body/tasksPage.jade

```diff
@@ -0,0 +1,15 @@
+.container.isic-tasks-container
```

Comment:
Oops, I meant the other way around: put the content of `taskPage.styl` under `taskPage.styl`.